v2/factions.json

```diff
@@ -1,84 +1,80 @@
 [
   {
     "code": "adam",
-    "color": "000000",
+    "description": "",
```

Comment:
This field is missing from the other mini-factions entirely. Should it be present as an empty string for them too?

Reply:
the schema allows it to be present or absent. I don't personally have strong feelings about requiring everything to be present even if the default is empty, but let me know if you do.

Reply:
I just noticed it was inconsistent. I don't mind either way.

Reply:
ok, let's leave off optional fields instead of having them present with an empty value.